metadata-ingestion/src/datahub/ingestion/source/unity/report.py

```diff
@@ -31,6 +31,8 @@ class UnityCatalogReport(IngestionStageReport, SQLSourceReport):
     tables: EntityFilterReport = EntityFilterReport.field(type="table/view")
     table_profiles: EntityFilterReport = EntityFilterReport.field(type="table profile")
     notebooks: EntityFilterReport = EntityFilterReport.field(type="notebook")
+    models: EntityFilterReport = EntityFilterReport.field(type="model")
```

Comment:
here as well can we name this to ml_models?

Reply:
updated